public/js/test/fixtures/todomvc.updateOnEnter.json

```diff
@@ -112,6 +112,28 @@
         "loading": false,
         "id": "server2.conn120.50",
         "text": "this.close();"
+      },
+      "server2.conn120.source42:142": {
+        "location": {
+          "sourceId": "server2.conn120.source42",
+          "line": 142
+        },
+        "condition": null,
+        "disabled": false,
+        "loading": false,
+        "id": "server2.conn120.50",
+        "text": "return undefined;"
+      },
+      "server2.conn120.source42:169": {
+        "location": {
+          "sourceId": "server2.conn120.source42",
+          "line": 169
+        },
+        "condition": null,
+        "disabled": true,
+        "loading": false,
+        "id": "server2.conn120.50",
+        "text": "var name = text.toLowerCase();"
```

Comment:
Nice. I assume you added these by hand. 

We generate the fixtures with cypress in the tests/integrations/fixtures tests. You can add two more breakpoints in the updateOnEnter test. 

Cypress requires an account to run so once you install it you'll need to fill out an adoptee form and then ping them in their gitter channel so you have early access. . https://gitter.im/cypress-io/cypress

This info should be in the readme :)